doc-content/enterprise-only/ai/ai-credit-card-con.adoc

```diff
@@ -0,0 +1,117 @@
+The financial industry uses pragmatic AI for decisioning in several areas. One area is credit card charge disputes. When a customer identifies an incorrect or unrecognized charge on a credit card bill, the customer can dispute the charge.  Human intervention in credit card fraud detection is required in some cases but the majority of reported credit card fraud can be completely or partially resolved with pragmatic AI.
+
+This case study is about the fictitious Fortress Bank, the bank's customer Joe, and the business process management (BPM) developer Michelle. First we will look at how the bank originally used AI through {PRODUCT} digital decisioning and then we will see how Michelle enhanced the decision model with a Predictive Model Markup Language (PMML) model created from machine learning.
+
+Machine learning models such as Tensorflow and R produce predictive models. You can save these predicitive models in an open standard such as PMML so that you can use the model with {PRODUCT} or other products that support the PMML standard.
+
+Fortress Bank uses {PRODUCT} on {OPENSHIFT} to develop and run the Fortress Bank decision services. {OPENSHIFT} is a cloud platform for developing and running containerized applications. It is designed to enable applications and the data centers that support them to expand from just a few machines and applications to thousands of machines that serve millions of clients. {PRODUCT} is the Red Hat middleware platform for creating cloud-native business automation applications and microservices. It enables enterprise business and IT users to document, simulate, manage, automate, and monitor business processes and decisions. {CENTRAL} is the {PRODUCT} dashboard.
+
+== Credit card dispute digital decisioning with {PRODUCT}
+Joe is a Fortress Bank customer. Each month, he logs in to the Fortress Bank web site to review all of the charges on his bill before he pays it. This month, Joe sees a transaction that he recognizes but the amount is incorrect. The vendor has charged him 77 dollars instead of 47 dollars. Joe selects the row that contains the incorrect item and clicks *Dispute*.
+
+image:ai/transactions-history.png[]
+
+This action initiates a series of questions about the dispute:
+
+. Why are you disputing these transactions?
+. Has your card been with you the entire time?
+. Is there anything else you would like to tell us about this dispute?
+
+After Joe answers the questions, the web site provides Joe with an incident number, `000004`.
+
+Fortress Bank must now decide whether to reimburse the disputed amount without investigation or whether to investigate the claim. Investigation requires more resources so automatic acceptance of the disputed amount costs the bank less in terms of resources. However, if the bank automatically accepts all disputed amounts the cost will ultimately be greater to the bank because of money paid for fraudulent claims. Someone or something must make the decision to investigate or not.
```

Comment:
```suggestion
Fortress Bank must now decide whether to reimburse the disputed amount without human investigation or whether to investigate the claim manually. Manual investigation requires more resources so automatic processing of the disputed amount costs the bank less in terms of human resources. However, if the bank automatically accepts all disputed amounts the cost will ultimately be greater to the bank because of money paid for fraudulent claims. Someone or something must make the decision to investigate or not.
```